content/applications/inventory_and_mrp/inventory/management/misc/owned_stock.rst

```diff
@@ -1,45 +1,106 @@
-===============================
-Manage Stock that you don't own
-===============================
+=================================================
+Consignment: Buy and Sell Stock Without Owning It
```

Comment:
Consignment: Buy and sell stock without owning it

*need to consider the capitalization rules